client/src/lib-app/validations/validations-factory.js

```diff
@@ -23,4 +23,4 @@ class ValidatorFactory {
     }
 }
 
-export default ValidatorFactory;
\ No newline at end of file
+export default ValidationFactory;
```

Comment:
This is braking the test, it doesn't work. Revert this line, change the name of the file to `validator-factory.js` and update it where necessary